chakra-ui/react broken dependencies

# 🐛 Bug report

> npm install @chakra-ui/react  fails with errors resolving dependencies

## 💥 Steps to reproduce

start with a new clean project from the template
> npx create-react-app my-app --template @chakra-ui
template installs as expected
dependency install fails when installing @chakra-ui/react
snip...

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   peer react@"17.0.1" from react-dom@17.0.1
npm ERR!   node_modules/react-dom
npm ERR!     react-dom@"^17.0.1" from the root project
npm ERR!     peer react-dom@">=16.8" from framer-motion@3.1.4
npm ERR!     node_modules/framer-motion
npm ERR!       peer framer-motion@">=3.0.0" from @chakra-ui/react@1.1.2
npm ERR!       node_modules/@chakra-ui/react
npm ERR!         @chakra-ui/react@"*" from the root project
npm ERR!       7 more (@chakra-ui/checkbox, @chakra-ui/menu, ...)
npm ERR!     2 more (@chakra-ui/portal, @chakra-ui/toast)
npm ERR!   react@"^17.0.1" from the root project
npm ERR!   49 more (@chakra-ui/react, @emotion/react, @emotion/styled, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0" from @reach/utils@0.11.0
npm ERR! node_modules/@reach/alert/node_modules/@reach/utils
npm ERR!   @reach/utils@"0.11.0" from @reach/alert@0.11.0
npm ERR!   node_modules/@reach/alert
npm ERR!     @reach/alert@"0.11.0" from @chakra-ui/toast@1.1.3
npm ERR!     node_modules/@chakra-ui/toast
npm ERR!       @chakra-ui/toast@"1.1.3" from @chakra-ui/react@1.1.2
npm ERR!       node_modules/@chakra-ui/react
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

snip....

## 🧐 Expected behavior

> Expected a fully usable base app to be available for learnig with 

## 🧭 Possible Solution
> I'm new to node hence starting with this template to see how it all hooks up

## 🌍 System information

| Software         | Version(s) |
| ---------------- | ---------- |
| Chakra UI           |      1.0      |
| Browser              |    firefox         |
| Operating System |     macOs 10.15.7 with homebrew installed and fully up to date    |

## 📝 Additional information

- Screenshots or code
- Notes or link to ideas

--- a/.changeset/little-tomatoes-clean.md
+++ b/.changeset/little-tomatoes-clean.md
@@ -1,0 +1,6 @@
+---
+"@chakra-ui/toast": patch
+---
+
+Updated peer dependency version of @reach/alert to fix installation of
+@chakra-ui with npm v7

--- a/packages/toast/package.json
+++ b/packages/toast/package.json
@@ -58,7 +58,7 @@
     "@chakra-ui/theme": "1.5.0",
     "@chakra-ui/transition": "1.0.7",
     "@chakra-ui/utils": "1.1.0",
-    "@reach/alert": "0.11.0"
+    "@reach/alert": "0.13.0"
   },
   "devDependencies": {
     "@chakra-ui/system": "1.2.1",

